scripts/aws/docker_prepare.sh

```diff
@@ -6,6 +6,7 @@ sudo apt install docker.io docker-buildx -y && sudo usermod -aG docker ${USER}
 if [[ ! $(groups "${USER}" | grep -q docker) ]]; then
   sudo groupadd docker
   sudo gpasswd -a ${USER} docker
+  sudo chmod 666 /var/run/docker.sock
```

Comment:
Why we need this change?

Reply:
Cuz if it's a  new docker environment, and our account is a non-root account, we need this permission for a non-root account and the account is in docker group.

Reply:
Currently, nope. This was modified at very early stage when I enabled this path. I could revert this line. This issue might has been fixed by other commit or code.